### PR TITLE
Fixing potential crash Backtrace #827

### PR DESCRIPTION
**Pull request by samcake**
This bug https://highfidelity.fogbugz.com/f/cases/12237/CRASH-in-Item-update-when-_payload-is-null
There is no real repro because in theory, a null Payload in a valid Render Item shouldn't happen.
I added a test before updating a render::Item to make sure it exists.

THis may be not enough to avoid the crash, time and backtrace will tell...

## TEST PLAN
There is no repro for that crash.
It shouldn't break anything.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7120/app_12382_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7120/HighFidelity-Beta-PR12382-d39ed574bd2b8ffff58b1b31b295a732ade7fabe.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7120/HighFidelity-Beta-PR12382-d39ed574bd2b8ffff58b1b31b295a732ade7fabe.exe

File: libraries/render/src/render/Scene.cpp [hunk 1]
```diff
@@ -355,13 +361,15 @@ void Scene::updateItems(const Transaction::Updates& transactions) {
                 auto newCell = _masterSpatialTree.resetItem(oldCell, oldKey, item.getBound(), updateID, newKey);
                 item.resetCell(newCell, newKey.isSmall());
             }
-        } else {
+        }
+        else {
```
Comment (ZappoMan) [inline libraries/render/src/render/Scene.cpp]: coding standard: brace placement! (visual studio is messing with you)

File: libraries/render/src/render/Scene.cpp [hunk 2]
```diff
@@ -355,13 +361,15 @@ void Scene::updateItems(const Transaction::Updates& transactions) {
...
             if (newKey.isSpatial()) {
                 _masterNonspatialSet.erase(updateID);
 
                 auto newCell = _masterSpatialTree.resetItem(oldCell, oldKey, item.getBound(), updateID, newKey);
                 item.resetCell(newCell, newKey.isSmall());
-            } else {
+            }
+            else {
```
Comment (ZappoMan) [inline libraries/render/src/render/Scene.cpp]: coding standard: brace placement! (visual studio is messing with you)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7129/app_12382_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7129/HighFidelity-Beta-PR12382-af2ca1e6e465e25922ca3736cddfc58f2c1968da.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7129/HighFidelity-Beta-PR12382-af2ca1e6e465e25922ca3736cddfc58f2c1968da.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7151/app_12382_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7151/HighFidelity-Beta-PR12382-b69d795efec74a181fb95671aed2ca1f402017ea.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7151/HighFidelity-Beta-PR12382-b69d795efec74a181fb95671aed2ca1f402017ea.exe

**Comment by samcake** [comment]
the same went as a hotfix to 63.4 that came back through the merge back

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7171/app_12382_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7171/HighFidelity-Beta-PR12382-113b036b0fc429e624992aa282e3477b28916e51.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7171/HighFidelity-Beta-PR12382-113b036b0fc429e624992aa282e3477b28916e51.exe

**Pull request closed without merging**